src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    private int getRandomLength() {
+        return (new Random().nextInt(20));
+    }
+
+    public Figure getRandomFigure() {
+        int getRandomIndex = new Random().nextInt(5);
```

Comment:
5 is a magic number
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names